assets/post.css

```diff
@@ -19,10 +22,10 @@ body {
   -moz-font-feature-settings: "kern" 1;
```

Comment:
this bit was in the original jekyll post.css file I didn't touch it but that `-ms...` looks like it adds IE support